aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/ConnectEvent.java

```diff
@@ -90,12 +90,37 @@ public static class ContactData implements Serializable {
         private String previousContactId;
 
         @JsonProperty("Queue")
-        private String queue;
+        private Queue queue;
 
         @JsonProperty("SystemEndpoint")
         private SystemEndpoint systemEndpoint;
     }
 
+
+    @Data
+    @Builder(setterPrefix = "with")
+    @NoArgsConstructor
+    @AllArgsConstructor
+    public static class Queue implements Serializable, Cloneable {
+        @JsonProperty(value = "ARN")
+        private String arn;
+        @JsonProperty(value = "Name")
+        private String name;
+        @JsonProperty(value = "OutboundCallerId")
+        private OutboundCallerId outboundCallerId;
+    }
+
+    @Data
+    @Builder(setterPrefix = "with")
+    @NoArgsConstructor
+    @AllArgsConstructor
+    public static class OutboundCallerId implements Serializable, Cloneable {
+        @JsonProperty(value = "Address")
```

Comment:
`@JsonProperty(value = "Address")` can be simplified in each case to `@JsonProperty("Address")`